### ISSUE TITLE
Inkscape's Tools Symbolic Icons

### ISSUE THREAD

**Issue by Fatih20** [elementary/icons#916]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Feature
**Is your feature request related to a problem? Please describe.**
All of the left-bar tool icons in Inkscape doesn't have symbolic icon in this set

**Describe the solution you'd like**
There should be one because it feels like there's a disconnect between the our icons and Inkscape's built-in symbolic icons.

**Existing work**
The align and distribute icons in this set have symbolic versions.

If this is a desirable thing to add, I'll start work on it.

**Comment by Brin-o** [comment]
I completely agree with this. Inkscape is one of the few linux vector tools we have, adding symbolic icons for it would do quite a lot for making the program feel more at home inside of elementary os.
